packages/material-ui/package.json

```diff
@@ -43,7 +43,7 @@
     "debounce": "^1.1.0",
     "deepmerge": "^2.0.1",
     "dom-helpers": "^3.2.1",
-    "hoist-non-react-statics": "^2.5.0",
+    "hoist-non-react-statics": "^3.0.0",
```

Comment:
It's increasing the bundle size, however, it's because they have added a new dependency: react-is. It's required for having `forwardRef` support.